css/fieldmanager-group-tabs.css

```diff
@@ -89,13 +116,14 @@
 
 .fm-wrapper .wp-tabs-panel > .fm-wrapper .fm-group-inner {
 	border-top: solid 1px #dfdfdf;
+	border-right: none;
+	border-bottom: none;
+	border-left: none;
+	-webkit-box-shadow: none;
```

Comment:
At some point it'd probably be nice to set up webpack for FM (including autoprefixer).